neo/public/javascripts/timeseries/controllers.js

```diff
@@ -70,6 +70,18 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
 
             var minDate = timeDimension.bottom(1)[0].time;
             var maxDate = timeDimension.top(1)[0].time;
+
+            chart.append('button')
+                .text("Reset")
+                .style("border", "none")
+                .style("background-color", "Transparent")
+                .style("color", "blue")
+                .style("position", "relative")
+                .style("bottom", "110px")

```

Comment:
is it possible to use the relative size? 
sometimes we need to show the demo on a projector which has lower resolution, it is better to use `%` or something. Or you can lower your resolution to see if it looks OK.